Scripts/test-streams.sh

```diff
@@ -3,6 +3,7 @@
 SCRIPT_NAME=$(basename "$0")
 SCRIPT_DIR=$(dirname "$0")
 
+Scripts/install-pkgx.sh
```

Comment:
This prevents the script from being run from any directory. We can assume that `install-pkgx.sh` is in the same directory, thus maybe something like

```suggestion
"$(dirname -- "$0")"/install-pkgx.sh
```

(there might be a better way to write it) would make it possible to run the script from everywhere.